contracts/lending/WasabiBNPL.sol

```diff
@@ -0,0 +1,178 @@
+// SPDX-License-Identifier: MIT
```

Comment:
add unit tests

Reply:
add new settable variable `address factory` that will be used when calling WasabiOption#mint